.dotnet/src/Custom/FineTuning/FineTuningJobHyperparameters.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace OpenAI.FineTuning;
+
+[CodeGenModel("FineTuningJobHyperparameters")]
+public partial struct FineTuningJobHyperparameters
+{
+    // constructor with two arguments
+    public FineTuningJobHyperparameters(BinaryData nEpochs, IDictionary<string, BinaryData> serializedAdditionalRawData = null)
+    {
+        NEpochs = nEpochs;
+        SerializedAdditionalRawData = serializedAdditionalRawData ?? new Dictionary<string, BinaryData>();
+    }
+
+    // constructor with just nEpochs
+    public FineTuningJobHyperparameters(BinaryData nEpochs)
+    {
+        NEpochs = nEpochs;
+    }
+
+    // open draft PR to get feedback
+    // re-add hyperparameters
+    [CodeGenMember("n_epochs")]
```

Comment:
`CodeGenMember` and other attributes will pick up on the processed name rather than the wire one; if you're not customizing the names, it should be identical (e.g. `[CodeGenMember("NEpochs")] public BinaryData NEpochs { get; }` -- if/when you *do* rename, it might not gracefully handle the comparison when there's a not a direct normalized match for "NEpochs" anymore.